utils/testman/tests/changeset_test.go

```diff
@@ -0,0 +1,178 @@
+package tests
+
+import (
+	"crypto/rand"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/acorn-io/manager/tests/e2e"
+	"github.com/acorn-io/manager/tests/helper"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func randToken(length int) (string, error) {
+	randBytes := make([]byte, length)
+
+	_, err := rand.Read(randBytes)
+	if err != nil {
+		return "", err
+	}
+
+	return hex.EncodeToString(randBytes), nil
+}
+
+func trimToDir(path, targetDir string) (string, error) {
```

Comment:
Could use `filepath.Dir` maybe here instead?

Reply:
The working directory is typically quite nested. I'm using this function to trim the path down the the absolute path to the `aws` repo directory.